packages/msw/src/index.ts

```diff
@@ -12,7 +12,11 @@ import { getMockDefinition, getMockOptionsDataOverride } from './mocks';
 
 const MSW_DEPENDENCIES: GeneratorDependency[] = [
   {
-    exports: [{ name: 'rest', values: true }],
+    exports: [
+      { name: 'http', values: true },
+      { name: 'HttpResponse', values: true },
+      { name: 'delay', values: true },
+    ],
```

Comment:
Don't you need to change the package.json in /packages/msw/src` ?

Reply:
Believe it or not, I don't think we do - it's not listed as a dependency inside of the package.json here. My suspicion is because we generate all of the code, there's no requirement to import the package itself because we aren't using any of it's methods. It's only being used once the files are generated.

```json
{
  "name": "@orval/msw",
  "version": "6.19.1",
  "license": "MIT",
  "main": "./dist/index.js",
  "types": "./dist/index.d.ts",
  "files": [
    "dist"
  ],
  "scripts": {
    "build": "tsup ./src/index.ts --target node12 --clean --dts",
    "dev": "tsup ./src/index.ts --target node12 --clean --watch src",
    "lint": "eslint src/**/*.ts"
  },
  "dependencies": {
    "@orval/core": "6.19.1",
    "cuid": "^2.1.8",
    "lodash.get": "^4.4.2",
    "lodash.omit": "^4.5.0",
    "openapi3-ts": "^3.0.0"
  }
}

```

Reply:
Understood what about the CUID issue mentioned in #855 with the deprecated package?

Reply:
Have now removed.